### DISCUSSION TITLE
en/tutorial/advanced/flux-controlnet-workflow-guide

### DISCUSSION THREAD

**Discussion by giscus** [comfyui-wiki/Comments#3] [Announcements]
# en/tutorial/advanced/flux-controlnet-workflow-guide

A step-by-step guide on how to use official Flux ControlNet models in ComfyUI

https://comfyui-wiki.com/en/tutorial/advanced/flux-controlnet-workflow-guide

**Comment by mike40033** [comment]
Hello! The workflow you get when you click "Download Full Version Workflow" seems to be a workflow for FLUX.1 Redux, not for FLUX.1 ControlNets

**Reply by comfyui-wiki** [reply]
Ok, I will check and try to fix it.

**Reply by comfyui-wiki** [reply]
I have checked it and uploaded a new file, but it may take some time to redeploy. Thanks for the feedback.

**Comment by mike40033** [comment]
Thank you!

**Comment by xuanwoa** [comment]
Hello!Where can I get DepthPreprocessor node。

**Reply by comfyui-wiki** [reply]
You may need  to install [comfyui_controlnet_aux](https://github.com/Fannovel16/comfyui_controlnet_aux). :)
